src/Components/Header/Header.css

```diff
@@ -0,0 +1,38 @@
+.header-container {
+  margin-top: 43px;
```

Comment:
- [ ] Here, instead of applying `margin-top`, apply height of the header and align all the items to the center vertically.